src/components/input/utc-date-input.tsx

```diff
@@ -17,12 +25,13 @@ export function UTCDateInput(
 ) {
     const [formValue, setFormValue] = useState(value as Date | undefined)
     const hiddenRef = useRef(null as HTMLInputElement | null)
-    const minDate = new Date() // the minimum date that can be selected
+    const dateNow = new Date()
+    const minDate = value === undefined ? dateNow : value > dateNow ? dateNow : value // the minimum date that can be selected
```

Comment:
Maybe I'm picky but this would read better with some line breaks between the cascading ternary operations

Reply:
You're right, I added some line breaks.